website/views/data.py

```diff
@@ -83,15 +85,32 @@ def corporation(request, player, game, corporation_slug):
 @render('game/players.html')
 @find_player_from_game_id
 @inject_game_into_response
-def players(request, game, player):
+@turn_by_turn_view
+def players(request, game, player, turn):
 	"""
 	Players data
 	"""
 
-	players = game.player_set.all().annotate(Count('share')).select_related('user').order_by('name')
-
+	players = game.player_set.all().order_by('pk')
+	corporations = list(game.corporation_set.all().order_by('pk'))
+	shares = Share.objects.filter(player__game=game, turn=turn).select_related('corporation', 'player')
+	corporations_shares = []
+	totals = []
+	for player in players:
+		total = 0
+		for corporation in corporations:
+			total += get_shares_count(corporation, player, shares)
+		totals.append({'player': player, 'total': total})
```

Comment:
let's only append the total value here, and send the player list to the tempalte